template/memory-cards.js

```diff
@@ -1,21 +1,35 @@
-// See cardset-example.js for examples
-
 var MemoryCards = (function() {
-	function Ctor() {
-		//...
-		this.values = function() {
+  var SevenWonders = [
+    ['Porcelain Tower of Nanjing', 1],
+    ['Nanjing, China', 1],
+    ['The Colosseum', 2],
+    ['Rome, Italy', 2],
+    ['The Great Wall of China', 3],
+    ['China', 3],
+    ['Stonehenge', 4],
+    ['Wiltshire, England', 4],
+    ['Hagia Sophia', 5],
+    ['Istanbul,Turkey', 5],
+    ['Leaning Tower of Pisa', 6],
+    ['Pisa, Italy', 6],
+    ['Catacombs of Kom el Shoqafa', 7],
+    ['Alexandria, Egypt', 7]
+  ];
 
-		}
+  function Ctor() {
 
-		this.match = function(a,b) {
+    this.values = function() {
+      return SevenWonders.slice();
+    };
 
-		}
+    this.match = function(pair1, pair2) {
+      return (pair1[1] === pair2[1]);
+    };
 
-		this.display = function(val) {
-			
-		}
-	}
-	//...
+    this.display = function(val) {
+      return val[0];
+    };
+  }
 
-	return Ctor;
+  return Ctor;
 })();
```

Comment:
Looking good! Your code is really clean and easy to read. Good indentation. Everything works great.